tck/cmd/server.go

```diff
@@ -15,12 +18,23 @@ func main() {
 
 	// Initialize the SDK service
 	sdkService := new(methods.SDKService)
+	accountService := new(methods.AccountService)
```

Comment:
We can have a lazier initialisation for these services. If this function is responsible for any service initialisation it would scale up in code and even guessing runtimes.